.github/workflows/functional-test.yml

```diff
@@ -47,17 +54,42 @@ jobs:
       with:
         distribution: 'temurin'
         java-version: '17'
+    - name: AVD cache
+      uses: actions/cache@v3
+      id: avd-cache
+      with:
+        path: |
+          ~/.android/avd/*
+          ~/.android/adb*
+        key: avd-${{ matrix.apiLevel }}
```

Comment:
Did this help to improve the speed?

Reply:
Looks like it does shave some minute or two, not so much though